modules/account-lib/src/coin/baseCoin/enum.ts

```diff
@@ -29,6 +29,10 @@ export enum TransactionType {
   StakingWithdraw,
   // Handle smart contract calls
   ContractCall,
+  // Wallet owner opt-in to receive this asset in his wallet
+  EnableAsset,
+  // Wallet owner opt-out to receive this asset in his wallet
```

Comment:
```suggestion
  // Wallet owner opt-out to stop receiving this asset in his wallet
```

Reply:
"opt-out to receive" is equivalent to "choose to stop receiving".